README.md

```diff
@@ -1 +1,59 @@
-# aws-centralized-monitoring
\ No newline at end of file
+# aws-centralized-monitoring
+
+## Table of Contents
+- [Instructions](#instructions)
+- [Support](#support)
+
+## Instructions
+### Why use this script?
+The primary reason to use this script is to enable centralized, multi-account monitoring in AWS. You may choose to do this via the AWS console, but the process involves multiple steps that can be prone to human error. Additionally, this process must be repeated in each AWS region you wish to collect metrics in. This script will automate the process and deploy the necessary resources in each region you specify (or all AWS default regions) for a quick and seamless onboarding to AWS Centralized CloudWatch Monitoring.
```

Comment:
I think this is a great intro to why a person would want automation for setting up multi-account monitoring. imo, it would be useful  to hyperlink to AWS's docs on multi-account monitoring so readers can learn more about the process which is being automated here.